modules/core/src/scenegraph/gltf/gltf-environment.js

```diff
@@ -0,0 +1,97 @@
+import GL from '@luma.gl/constants';
+import {Texture2D, TextureCube} from '../../webgl';
+import {loadImage} from '../../core/load-file';
+
+export default class GLTFEnvironment {
+  constructor(gl, baseUrl) {
+    this.gl = gl;
+    this.baseUrl = baseUrl;
+  }
+
+  getTexUrl(envMap, type, dir, mipLevel = 0) {
+    return `${this.baseUrl}${envMap}/${type}/${type}_${dir}_${mipLevel}.jpg`;
+  }
+
+  getBrdfUrl() {
+    return `${this.baseUrl}brdfLUT.png`;
+  }
+
+  makeCube({id, getTextureForFace, parameters}) {
+    return new TextureCube(this.gl, {
+      id,
+      mipmaps: false,
+      parameters,
+      pixels: {
+        [GL.TEXTURE_CUBE_MAP_POSITIVE_X]: getTextureForFace('right'),
```

Comment:
Are they always called right/left etc or does that just happen to be the case for the example you had? Should this be a parameter to the class?

Reply:
User should subclass if they want different names.

Reply:
I'm gonna refactor this to accept **getTexUrl** as a function parameter. Then the user can do what they want.

Reply:
> Refactor this to accept getTexUrl as a function parameter

Awesome, thank you!

* Can we send the GL constant to that function, rather than the arbitrary (left/top/bottom...) strings? 
* I have seen a couple of conventions for this (`negative-x`, ...), not convinced we should choose one on the user's behalf, unless you have more evidence that this being standardized.

Reply:
I'll use the gl constant.

Reply:
Done